src/providers/MeetingProvider/MeetingManager.ts

```diff
@@ -223,10 +185,15 @@ export class MeetingManager implements AudioVideoObserver {
     configuration: MeetingSessionConfiguration,
     deviceLabels: DeviceLabels | DeviceLabelTrigger = DeviceLabels.AudioAndVideo,
   ): Promise<any> {
-    const logger = this.logger ? this.logger : this.createLogger(configuration);
+    const logger = this.config.logger ? this.config.logger : this.createLogger(configuration);
```

Comment:
You can avoid `this.config.x` each time by using the object restructuring first.
```js
const { logger, simulcastEnabled, videoDownlinkBandwidthPolicy, postLogConfig } = this.config;
```

Reply:
oh, that's right!  good suggestions